applications/nrf_desktop/README.rst

```diff
@@ -339,7 +339,9 @@ The project comes with configuration files for the following boards:
     * PCA20037 - nRF Desktop keyboard reference design
     * PCA20044 - nRF Desktop casual mouse reference design (nRF52832)
     * PCA20045 - nRF Desktop casual mouse reference design (nRF52810)
-    * PCA10059 - nRF Desktop dongle reference design
+    * PCA10059 - nRF Desktop dongle reference design (nRF52840)
```

Comment:
PCA10059 is the nRF52840 Dongle, not a reference design.
So maybe this needs to be worded similar to line 337?

Reply:
This is actually working HID dongle - from application perspective. It is not emulated but rather shares the same HW platform. On the left is name of the board on the right is what application does with it.

Reply:
This must be worded differently then, because right now you're saying that PCA10059 is the nRF Desktop dongle reference design.
Also, with regards to the board naming discussion (NCSDK-4678), we should probably start using product names (thus nRF52840 Dongle) instead of PCA numbers here.

Reply:
You are right that the sentence is probably confusing. Regarding the naming this is much more work then it seems and we cannot change it in one place only.